PhantomIntelligenceHeader header
int16[] data
